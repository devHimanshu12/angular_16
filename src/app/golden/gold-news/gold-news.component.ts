import { Component } from '@angular/core';

@Component({
  selector: 'app-gold-news',
  templateUrl: './gold-news.component.html',
  styleUrls: ['./gold-news.component.scss']
})
export class GoldNewsComponent {

}
