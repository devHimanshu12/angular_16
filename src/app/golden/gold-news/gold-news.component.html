<p>gold-news works!</p>
